hotel_housekeeping/__manifest__.py

```diff
@@ -38,7 +36,6 @@
         "report/hotel_housekeeping_report.xml",
         "views/activity_detail.xml",
         "wizard/hotel_housekeeping_wizard.xml",
-        "views/hotel_housekeeping_workflow.xml",
```

Comment:
You removed the workflow but made sure to add the same functionality by object and buttons?